examples/quickstart.ipynb

```diff
@@ -622,51 +629,21 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
    "source": [
     "Note: the small object size is a consequence of the empty sphere around the cube."
    ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Passing a different mesh\n",
```

Comment:
I would consider keeping this section, shorten the content to the required minimum and point to the meshes notebook.